Update french.xml to 7.7.2

xylographe opened the pull request:
- Add missing entries.
- Remove surplus entries.
- Modify encoding menu entries according to #5951.

donho commented:
```
$ git log
commit c52e917e515c6ace34b95e96ca021d502cd77c82 (HEAD -> master)
Author: Waldi Ravens <wr86420@gmail.com>
Date:   Wed Jul 31 04:02:46 2019 +0200

    Update french.xml to 7.7.2

    - Add missing entries.
    - Remove surplus entries.
    - Modify encoding menu entries according to #5951.

commit 12ca892176745e7bf2370ec561942dc0372a885d
Author: Waldi Ravens <wr86420@gmail.com>
Date:   Wed Jul 31 04:02:46 2019 +0200

    Update french.xml to 7.7.2

    - Add missing entries.
    - Remove surplus entries.
    - Modify encoding menu entries according to #5951.

commit 99c8610774a8b0342f4d9bd9cf4060453d62136e (origin/master, origin/HEAD)
Author: bovirus <1262554+bovirus@users.noreply.github.com>
Date:   Tue Jul 30 17:38:23 2019 +0200

    Update italian.xml

    Close #5977
```

donho commented:
@xylographe 
The above is the git log displayed info after merging your PR.

Please make sure you your master is updated with origin (notepad-plus-plus/notepad-plus-plus
) by using `git pull` then create a new branch (with new branch name) where you apply your modification.

In this way, your PR will contain a clean git history that can be merged.

xylographe commented:
@donho
I'm afraid, I don't understand what is going on here,
```
PS> git fetch -v upstream master
From https://github.com/notepad-plus-plus/notepad-plus-plus
 * branch              master     -> FETCH_HEAD
 = [up to date]        master     -> upstream/master
PS> git checkout master
Switched to branch 'master'
Your branch is up to date with 'upstream/master'.
PS> git checkout pr/190731/update_french
Switched to branch 'pr/190731/update_french'
Your branch is up to date with 'origin/pr/190731/update_french'.
PS> git log -n2
commit 5efe1a0f4d5bb24c10510bd8aacd0da5a62c96fb (HEAD -> pr/190731/update_french, origin/pr/190731/update_french)
Author: Waldi Ravens <wr86420@gmail.com>
Date:   2019-07-31 04:02:46 +0200

    Update french.xml to 7.7.2

    - Add missing entries.
    - Remove surplus entries.
    - Modify encoding menu entries according to #5951.

commit 99c8610774a8b0342f4d9bd9cf4060453d62136e (upstream/master, upstream/HEAD, origin/master, master)
Author: bovirus <1262554+bovirus@users.noreply.github.com>
Date:   2019-07-30 17:38:23 +0200

    Update italian.xml

    Close #5977
```
As you can see my local `master` branch and the `upstream/master` branch are synchronized. The commit in the `pr/190731/update_french` branch has commit-id `5efe1a0f4d5bb24c10510bd8aacd0da5a62c96fb`, which is different from the two commit-ids in your `git log` output above.

If you visit commit 5efe1a0f4d5bb24c10510bd8aacd0da5a62c96fb on GitHub, you will see that the parent is commit `99c8610774a8b0342f4d9bd9cf4060453d62136e` (Update italian.xml). Furthermore, the [`pr/190731/update_french` branch on GitHub](https://github.com/xylographe/notepad-plus-plus/tree/pr/190731/update_french) shows “This branch is 1 commit ahead of notepad-plus-plus:master.”, and the _Latest commit_ on that branch is  `5efe1a0f4d5bb24c10510bd8aacd0da5a62c96fb`, and its _parent_ is commit `99c8610774a8b0342f4d9bd9cf4060453d62136e`.

I will create a new PR, but I wonder if that will resolve the problem.
